doc/sphinx/source/notes_7_0.rst

```diff
@@ -21,9 +38,58 @@ CPU drivers
   Please note that these DBKs are still a proof-of-concept and
   are subject to change without notice.
 
+* When built with -DENABLE_CONFORMANCE=ON, PoCL's CPU driver
+  is now able to pass the full OpenCL-CTS conformance testsuite.
+  We have submitted & achieved conformant status on <TODO day>.
+
+* Implemented Windows platform support, using both MinGW and MSVC
+  toolchains.
+
+* Support for generating NSIS package on Windows
+
 * cl_khr_command_buffer implementation has been updated to 0.9.6
   Note that this version is not backwards compatible with previous versions.
 
+* implemented extension cl_ext_buffer_device_address / 1.0.2
```

Comment:
Bullet point sentences should start with a cap. If one ends in a dot, all should.

Reply:
fixed